contracts/base/LoansBase.sol

```diff
@@ -494,10 +494,57 @@ contract LoansBase is LoansInterface, Base {
         address payable recipient
     ) internal {
         if (loans[loanID].escrow != address(0x0)) {
-            EscrowInterface(loans[loanID].escrow).claimTokens(recipient);
+            /**
+                - Check the value of the liquidator will get as a reward - total amount owed * 1.05 and converted into collateral asset 
+                settings().chainlinkAggregator().valueFor(
+                    baseAddress,
+                    quoteAddress,
+                    baseAmount
+                ); */
+            uint256 totalOwed = _getTotalOwed(loanID);
+            uint256 amountToLiquidatorInLendingTokens = totalOwed.mul(105).div(100);
```

Comment:
Make sure to use the `LIQUIDATE_ETH_PRICE` value. You can also use the `NumbersLib.percent()` function.

Reply:
Using LIQUIDATE_ETH_PRICE_SETTING